db/dcrsqlite/apisource.go

```diff
@@ -1262,3 +1262,17 @@ func (db *wiredDB) CountUnconfirmedTransactions(address string, maxUnconfirmedPo
 	}
 	return
 }
+func (db *wiredDB) GetNumberOfTicketsPerBlock(idx int) (numberOfTicketsPerBlock int64, err error) {
```

Comment:
Newline and docs please

Reply:
Just `NumberOfTicketsPerBlock` please.  We need to change them all in the future.

Reply:
thinking of removing the function entirely, It currently serves no purpose if there exist this exp.ChainParams.TicketsPerBlock already